PowerPointLabs/PowerPointLabs/HighlightBulletsText.cs

```diff
@@ -99,7 +99,10 @@ public static void AddHighlightBulletsText()
                     isFirstShape = false;
                 }
 
-                Globals.ThisAddIn.Application.CommandBars.ExecuteMso("AnimationPreview");
+                if (currentSlide.HasAnimationForClick(1))

```

Comment:
can u help refactor the param of `Slide.HasAnimationForClick` from `click` to `clickNumber`,
and here add a prefix `clickNumber: ` for the magic number